commcare_connect/opportunity/tasks.py

```diff
@@ -227,3 +231,31 @@ def send_sms_task(user_ids: list[int], body: str):
     user_phone_numbers = User.objects.filter(id__in=user_ids).values_list("phone_number", flat=True)
     for phone_number in user_phone_numbers:
         send_sms(phone_number, body)
+
+
+@celery_app.task()
+def download_user_visit_attachments(user_visit_id: id):
+    user_visit = UserVisit.objects.get(id=user_visit_id)
+    api_key = user_visit.opportunity.api_key
+    blobs = user_visit.form_json.get("attachments", {})
+    domain = user_visit.opportunity.deliver_app.cc_domain
+    form_id = user_visit.xform_id
+    for name, blob in blobs.items():
+        if name == "form.xml":
+            continue
+        url = f"{settings.COMMCARE_HQ_URL}/a/{domain}/api/form/attachment/{user_visit.xform_id}/{name}"
+
+        blob_meta = BlobMeta(
```

Comment:
nit: This can be replaced with the `BlobMeta.objects.get_or_create` method which internally executes the same code and returns the `object` and a `created` flag which can be used to continue the loop when the created flag is false.